src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -685,12 +692,24 @@ private EndpointBitrateAllocation(
 
             // Initialize rates.
             rates = new long[encodings.length];
+            int optimalThumbnailIndex = 0;
             for (int i = 0; i < encodings.length; i++)
             {
                 rates[i] = encodings[i].getLastStableBitrateBps();
+                if (encodings[i].getHeight() <= THUMBNAIL_MAX_HEIGHT)
+                {
+                    optimalThumbnailIndex = i;
+                }
             }
 
-            optimalIdx = selected ? encodings.length - 1 : (forwarded ? 0 : -1);
+            // TODO Determining the optimal index needs some work. The optimal
```

Comment:
I'm not sure we'd ever want to do that. Mobile devices these days have so many pixels that a 360p video would look terrible.